charts/grafana/values.yaml

```diff
@@ -124,6 +124,19 @@ service:
   labels: {}
   portName: service
 
+serviceMonitor:
+  ## If true, a ServiceMonitor CRD is created for a prometheus operator
+  ## https://github.com/coreos/prometheus-operator
+  ##
+  enabled: false
+  path: /metrics
+  #  namespace: monitoring
```

Comment:
Can we set this to `namespace: ""` by default and maybe add a comment above what it does? Or which namespace is used by default.

Reply:
> 
> 
> Can we set this to `namespace: ""` by default and maybe add a comment above what it does? Or which namespace is used by default.

@torstenwalter - good catch, thank you.  I've updated the readme to reflect the additions to values.yaml with a description and default value.  If the documentation doesn't make sense please let me know and I can update the descriptions.